src/ImageProcessorCore/IO/EndianBinaryReader.cs

```diff
@@ -122,7 +122,7 @@ public void Close()
         /// </summary>
         /// <param name="offset">Offset to seek to.</param>
         /// <param name="origin">Origin of seek operation.</param>
-        public void Seek(int offset, SeekOrigin origin)
+        public virtual void Seek(int offset, SeekOrigin origin)
```

Comment:
It's easy to do. I'm currently making an absolute meal of converting the gif encoder to use the EndianBinaryWriter. Gonna have to break out the hex editor!